hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -1212,6 +1218,13 @@
       en_run_modes: ["sw_test_mode_test_rom"]
       run_opts: ["+sw_test_timeout_ns=20_000_000"]
     }
+    {
+      name: chip_sw_keymgr_sideload_aes
+      uvm_test_seq: chip_sw_keymgr_sideload_aes_vseq
+      sw_images: ["//sw/device/tests:aes_sideload_test:1"]
+      en_run_modes: ["sw_test_mode_test_rom"]
+      run_opts: ["+sw_test_timeout_ns=20_000_000"]
+    }
```

Comment:
with this, we could remove the test `chip_sw_aes_sideload` in line 995

Maybe also rename `aes_sideload_tes.c` to `keymgr_sideload_aes_test.c` for the sake of consistency

Reply:
that makes sense. did that in my last push.